packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testRedoStructureUnapplyUnappliesStructureWith..st

```diff
@@ -0,0 +1,9 @@
+tests
+testRedoStructureUnapplyUnappliesStructureWith: aBlock
+
+	self 
+		setupBoldText;
+		deselectOptionBold.
+	self editorModel undo.
+	aBlock value.
```

Comment:
Perhaps name it `redoBlock` as you did in `testRedoStructureWith` to make clear what the block is for

Reply:
Does this really belong into the tests category as it is only an utilily for acceptance tests as far as I understood?